src/cosl/coordinated_workers/worker.py

```diff
@@ -406,6 +406,12 @@ def _add_readiness_check(self, new_layer: Layer):
 
     def _reconcile(self):
         """Run all unconditional logic."""
```

Comment:
funny that this says - run all unconditional logic - and the first thing we see two lines below is... a conditional! perhaps rephrase it to say something a bit more precise like "logic that does not depend on what event we're processing"

Reply:
updated docstring